BetterUI/Main.cs

```diff
@@ -33,10 +33,12 @@ public const string
         public static ConfigEntry<KeyCode> modKeySecondary;
         public static ConfigEntry<bool> useCustomHealthBar;
         public static ConfigEntry<bool> useCustomStaminaBar;
+        public static ConfigEntry<bool> useCustomSpoilerBar;
         public static ConfigEntry<bool> useCustomFoodBar;
-        public static ConfigEntry<int> healthBarRotation;
-        public static ConfigEntry<int> staminaBarRotation;
-        public static ConfigEntry<int> foodBarRotation;
+        public static ConfigEntry<int> customHealthBarRotation;
```

Comment:
I get what you're trying to do here, but is it worth resetting the current values everyone has set?

Reply:
as mentioned, changing these to a different section would reset the settings anyway. if we don't move them to a different section, then of course we shouldn't rename the rotation config options. but I thought if we reset them with the section change anyway, then now is a good name to improve the name